test/commands.test.ts

```diff
@@ -295,7 +299,17 @@ suite('Command Handler', () => {
                 expect(err).equals(errorStatus.message);
             }
         });
-    });
+
+        test('wont remove if user does not confirm', async () => {
+          sandbox.stub(vscode.window, 'showWarningMessage').resolves();
+          try {
```

Comment:
No need for try-catch here, it's just catching the arbitrary expect.fail anyway.